aufgabe/FünfPunktEins.java

```diff
@@ -0,0 +1,18 @@
+public class FünfPunktEins {
+    public static void main(String[] args) {
+        for (int i=1; i<=100; i++){
```

Comment:
autoformatting 😈 

in der Java welt setzen wir geschwungene klammern nicht auf eine eigene zeile 

d.h. 
`} else if (...) {`